Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -387,9 +387,10 @@ final class StatusTableViewController: LoopChartsTableViewController {
                     }
                     reloadGroup.leave()
                 }
-                carbsOnBoard = state.carbsOnBoard?.quantity
             }
-
+            // always check for cob
+            carbsOnBoard = state.carbsOnBoard?.quantity
```

Comment:
What was causing cob to update without .carbs being in the refresh context?

Reply:
It isn't updating, but that `carbsOnBoard` is set to `nil` with each reload, which causes the cob string to also be set to `nil` until carbs are entered into loop.

Are you suggesting that cob should be a stored property of the VC?

Reply:
Ok, I see this is a new var that was added. Sounds good.